src/frontend/views/Github/GithubTemplate.tsx

```diff
@@ -0,0 +1,171 @@
+import { AttestationProcessAnchoring } from '../../components/AttestationProcessAnchoring/AttestationProcessAnchoring';
+import { AttestationProcessReady } from '../../components/AttestationProcessReady/AttestationProcessReady';
+import { SigningErrorClosed } from '../../components/SigningErrorClosed/SigningErrorClosed';
+
+import { GithubProfile } from './Github';
+
+export type AttestationStatus =
+  | 'none'
+  | 'urlReady'
+  | 'authorizing'
+  | 'authorized'
+  | 'attesting'
+  | 'ready'
+  | 'error';
+
+export type FlowError = 'unauthorized' | 'closed' | 'unknown';
+
+interface Props {
+  status: AttestationStatus;
+  processing: boolean;
+  handleSubmit: FormEventHandler;
+  handleBackup: MouseEventHandler;
+  handleTryAgainClick: MouseEventHandler;
+  authUrl?: string;
+  flowError?: FlowError;
+  profile?: GithubProfile;
+}
+export function GithubTemplate({
+  status,
+  processing,
+  handleSubmit,
+  handleBackup,
+  handleTryAgainClick,
+  authUrl,
+  flowError,
+  profile,
+}: Props): JSX.Element {
+  const preventNavigation = usePreventNavigation(
+    processing || ['authorizing', 'authorized', 'attesting'].includes(status),
+  );
+
+  return (
+    <section
+      className={cx(flowStyles.container, {
+        [flowStyles.processing]: processing,
+      })}
+      aria-busy={processing}
+    >
+      {processing && <Spinner />}
+
+      <h1 className={styles.heading}>Github Account Attestation</h1>
+
+      <Prompt
+        when={preventNavigation}
+        message="The Github attestation process has already started. Are you sure you want to leave?"
+      />
+
+      <Explainer>
+        After you sign into your GitHub account and give SocialKYC permission,
+        SocialKYC requests your GitHub information for the credential. You can
+        then sign the data with one of your identities in Sporran, and SocialKYC
+        will create the credential.
+      </Explainer>
+
+      {status === 'none' && (
+        <DetailedMessage
+          icon="spinner"
+          heading="Attestation process:"
+          message="Generating link"
+          details="Please wait, generating GitHub authorization link."
+        />
+      )}
+
+      {status === 'urlReady' && (
+        <p className={styles.buttonsLine}>
+          <a
+            className={styles.ctaButton}
+            href={authUrl}
+            target="_blank"
+            rel="noreferrer"
+          >
+            Sign in with GitHub
+          </a>
+        </p>
+      )}
+
+      {status === 'authorizing' && (
+        <DetailedMessage
+          icon="spinner"
+          heading="Attestation process:"
+          message="Authorizing"
+          details="Please wait, accessing Github account details."
+        />
+      )}
+
+      {status === 'authorized' && profile && (
+        <form onSubmit={handleSubmit}>
+          <dl className={styles.profile}>
+            <dt>User-ID:</dt>
+            <dd>{profile.id}</dd>
+
+            <dt>Username:</dt>
```

Comment:
Should this be called `Login` as well?

Reply:
I guess this is a question for the design team/wording decision makers

Reply:
Personally I don't really like GitHub's choice here. I would say Username is clearer. But if we want to preserve the data as we receive it then I guess it should be Login.

Reply:
We have already put it in the CType as Login

Reply:
Interesting that even GitHub themselves call this "Username" on their sign in screen

Reply:
I would prefer to change it to Username everywhere, since GitHub does the same. What do you think?

Reply:
Okay, go for it!